### ISSUE TITLE
AWS Glue Crawlers Do Not Support SecurityConfiguration

### ISSUE THREAD

**Issue by matthewhenricks** [terraform-providers/terraform-provider-aws#6714]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

AWS Glue Crawlers don't appear to have the SecurityConfiguration settable in Terraform. The boto3 documentation specifies the following:

`
response = client.create_crawler(
    Name='string',
    ...
    CrawlerSecurityConfiguration='string'
)
`

CrawlerSecurityConfiguration (string) -- The name of the SecurityConfiguration structure to be used by this Crawler.

The Terraform Documentation does not include support for this field - making running a crawler potentially insecure.

https://www.terraform.io/docs/providers/aws/r/glue_crawler.html

### New or Affected Resource(s)

* aws_glue_crawler

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.create_crawler
* https://www.terraform.io/docs/providers/aws/r/glue_crawler.html

**Comment by ilya-v-trofimov** [comment]
I submitted a PR which adds `security_configuration` argument to the `aws_glue_crawler`, that should address this issue

**Comment by bflad** [comment]
Support for a new `security_configuration` argument has been merged and will release with version 1.52.0 of the AWS provider, likely later today or tomorrow. 👍

**Comment by bflad** [comment]
This has been released in [version 1.52.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1520-december-13-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
